src/container.c

```diff
@@ -13,6 +13,8 @@
 #include <time.h>
 #include <unistd.h>
 
+char *join_strings(const char *strings[], int start, int end);
```

Comment:
Is it any string or the arguments for the container? If the latter, we might want to rename the function to reflect that.